### DISCUSSION TITLE
FLRC modulation support for SX1280

### DISCUSSION THREAD

**Discussion by buhalo1** [markqvist/Reticulum#457] [Ideas]
I'm very happy that RNode now supports SX1280 modems. I want to try to use these modems for linking amateur radio repeaters using asterisk and the app_rpt functionality built into it. I want to try linking repeaters by IP or transmitting the iax2 stream in transparent mode using a proxy server.
I hope that these modems will allow to setup a longer and more stable links in comparison with wifi bridges.
Theoretically, the throughput when using lora modulation is sufficient for some options for organizing such links, but FLRC opens up unlimited possibilities for such applications. It would be very useful to be able to use this kind of modulation in RNode.

**Comment by markqvist** [comment]
Indeed, FLRC modulation support would be great.

**Reply by jacobeva** [reply]
Yes. It is something I thought of when writing the driver! On the wishlist certainly ;)
